packages/cw-orch-osmosis-test-tube/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "cw-orch-osmosis-test-tube"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+# Test Tube env deps
+osmosis-test-tube = { workspace = true }
+prost-types = { workspace = true }
+prost = { workspace = true }
+cosmwasm-std.workspace = true
+serde.workspace = true
+cw-orch-core = { workspace = true }
+cw-orch-mock = { workspace = true }
```

Comment:
It needs mock for the state struct?

Reply:
Maybe state should be its own package so it can be versioned / developed independently? Might not be worth the hassle